_redirects

```diff
@@ -6,8 +6,6 @@ layout: null
 / /docs/ 301!
 /docs/v2.2/* /docs/v19.1/:splat 301!
 /docs/managed/* /docs/cockroachcloud/:splat 301!
-/docs/releases/ /docs/releases/index.html 301!
-/docs/advisories/ /docs/advisories/index.html 301!
```

Comment:
Just leaving a comment per our discussion: these redirects may be used by downstream consumers, TSE org, etc. As long as we are OK with potential downstream impact here (we can't know who uses redirects, for example!), then this LGTM. Understand the SEO gains from this now, thanks for the helpful explanations!